documentation/release_notes.rst

```diff
@@ -8,6 +8,65 @@ The Intel® oneAPI DPC++ Library (oneDPL) accompanies the Intel® oneAPI DPC++/C
 and provides high-productivity APIs aimed to minimize programming efforts of C++ developers
 creating efficient heterogeneous applications.
 
+New in 2022.3.0
+===============
+
+New Features
+------------
+- Added dynamic selection as an experimental feature. Provides functions such as ``select``,
+  ``submit`` and ``submit_and_wait``. It includes selection policies: ``fixed_resource_policy``,
+  ``round_robin_policy``, ``dynamic_load_policy``, and ``auto_tune_policy``. And provides default
+  support for SYCL queues. 
+-	Algorithms using vector execution policies are now vectorized whenever possible when the program is built with Intel® DPC++/C++ Compiler.
+- Added support for passing zip iterators as segment value data in ``reduce_by_segment``, ``exclusive_scan_by_segment``,
+  and ``inclusive_scan_by_segment``.
+- Improved performance of ``merge``, ``sort`` and ``stable_sort`` (non-Radix case) algorithms on GPU devices.
+  Added new implementation based on ``Merge Path`` approach.
+- Improved performance of ``sort`` and ``sort_stable`` (Radix case) algorithms on GPU devices.
+- Improved performance of the ``reduce``, ``min_element``, ``max_element``, ``minmax_element``,
+  ``is_partitioned``, and ``lexicographical_compare`` algorithms with DPC++ execution policies.
+- When compiling with DPC++ compiler option, libstdc++ version 8 or libc++, and with ``-fsycl-pstl-offload`` option,
+  parallel algorithms from std namespace called with ``oneapi::dpl::parallel_unsequenced_policy`` would be
+  offloaded to the SYCL device in accordance with ``-fsycl-pstl-offload`` option value.
```

Comment:
@kboyarinov, @akukanov, @Alexandr-Konovalov, @rarutyun would you prefer to describe this change in a separate section?

Reply:
Moved this change to "Known Issues and Limitations" section.